build/win/generate_winrt_headers.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import os
+import sys
+
+def clean(output_dir):
+  if os.path.exists(output_dir):
+    os.remove(output_dir)
+  return
+
+
+def generate_headers(output_dir):
+  """Run cppwinrt.exe on the installed Windows SDK version and generate
+  cppwinrt headers in the output directory.
+  """
+  
+  cppwinrt_exe = os.path.join(
+  __file__,
+  '..\\..\\third_party\\cppwinrt\\bin\\cppwinrt.exe')
+
+  args = [cppwinrt_exe, '-in', 'sdk',
+      '-out', '%s' % output_dir]
+
+  return 0

```

Comment:
Looks like we're not currently running the command here. Does this need a re-push?

Reply:
Might also be a good idea for us to first recursively delete the output directory (if it exists) then run the command.

Reply:
Good catch ;-)  I think i fat fingered the porting from my test branch.  On the delete front, I have the Clean() method that is called before `generate_headers()` but that isn't do a recursive.  I'll take another look to make sure that's working as expected

Reply:
Both fixed in most recent commit